testdata/scripts/debugdir.txt

```diff
@@ -1,11 +1,21 @@
 env GOPRIVATE=test/main
+# Isolate go cache for test reproducibility
+env GOCACHE=$WORK/.temp-cache
```

Comment:
don't do this, you especially want `go test` to reuse caches as usual. your test should still pass. if every file started with an empty cache it would be more "pure", but much slower.